cli/info/info.go

```diff
@@ -44,9 +47,35 @@ func infoMain(args []string, c cli.Config) (err error) {
 	if err != nil {
 		return err
 	}
+
+	_, err = helpers.ParseCertificatePEM(certPem)
+	if err != nil {
+		return errors.Wrap(errors.CertificateError, errors.DecodeFailed, err)
+	}
+
+	s, err := sign.SignerFromConfig(c)
```

Comment:
I see. For some reason, the map in Server.Req only had "certificate" as a key before, so I was kind of floundering to get the other data. This should be no problem.